string name
uint32 state_machine_id
